packages/@react-spectrum/s2/src/style-utils.ts

```diff
@@ -131,6 +131,8 @@ export const fieldInput = () => ({
 
 export const colorScheme = () => ({
   colorScheme: {
+    // Default to page color scheme if none is defined.
+    default: 'var(--lightningcss-light, light) var(--lightningcss-dark, dark)',
```

Comment:
Don't want to define the browser `color-scheme` property outside our provider since it might affect other content, so define it here based on the variables above which will be either defaulted or given by page.css.